contributing-docs/03_contributors_quick_start.rst

```diff
@@ -47,7 +47,7 @@ Before deciding which method to choose, there are a couple of factors to conside
 Local machine development
 #########################
 
-If you do not work in remote development environment, you need those prerequisites.
+If you do not work in remote development environment, you need these prerequisites.
```

Comment:
nit: If you do not work in remote development environments, you will need these prerequisites: